### DISCUSSION TITLE
building game error

### DISCUSSION THREAD

**Discussion by Ibexlol** [KadeArchive/Kade-Engine#2112] [Q&A]
im trying to build my game but it says "C:/HaxeToolkit/haxe/lib/extension-webm/git/ndll/Windows64/extension-webm.ndll" and i have extension-webm  installed already please help

**Comment by NK-523** [comment]
are there any other errors?

**Reply by Ibexlol** [reply]
No

**Reply by NK-523** [reply]
try running this (if you haven't already) `lime rebuild extension-webm <ie. windows, macos, linux>` in command prompt. If that doesn't fix it, try running `haxelib remove extension-webm` then run `haxelib git extension-webm https://github.com/KadeDev/extension-webm`.
